instrumentation/sidekiq/lib/opentelemetry/instrumentation/sidekiq/middlewares/client/tracer_middleware.rb

```diff
@@ -14,10 +14,13 @@ module Client
           class TracerMiddleware
             def call(_worker_class, job, _queue, _redis_pool)
               tracer.in_span(
-                job['wrapped']&.to_s || job['class'],
+                span_name(job),
                 attributes: {
+                  'messaging.system' => 'sidekiq',
+                  'messaging.sidekiq.job_class' => job['wrapped']&.to_s || job['class'],
                   'messaging.message_id' => job['jid'],
-                  'messaging.destination' => job['queue']
+                  'messaging.destination' => job['queue'],
+                  'messaging.destination_type' => 'queue'
```

Comment:
```suggestion
                  'messaging.destination_kind' => 'queue'
```
Not sure why we had `destination_type`. Please do re-check the semantic conventions in case my eyes are playing tricks on me.

Reply:
Woops, you're right, conventions call for `_kind` not `_type`